frontend/src/components/client/EditClient.tsx

```diff
@@ -106,6 +107,12 @@ const EditClient = () => {
         <Row>
           <h1 style={{marginBottom: 10}}>
             {client.name || (initClient._id ? '' : t('client.createNew'))}
+            <NotesWithCommentsModalButton
+                 claim={Claim.ManageClients}
+                 value={{note: client.notes, comments: client.comments }}
+                 onChange={val => setClient( { ...client, notes: val.note || '', comments: val.comments} )}
+                 title={t('client.comments')}
+                 style={ {marginLeft: 6, marginBottom: 6}} />
```

Comment:
De .editorconfig was nu toch installed?
Ik zie hier nu 4 spaces indenting ipv 2

Normaal gezien is de layout ook altijd ala:

```
<NotesWithCommentsModalButton
                 claim={Claim.ManageClients}
                 value={{note: client.notes, comments: client.comments }}
                 onChange={val => setClient( { ...client, notes: val.note || '', comments: val.comments} )}
                 title={t('client.comments')}
                 style={ {marginLeft: 6, marginBottom: 6}}
/>
```

Die zorgt voor een kleinere diff wanneer je een property toevoegt of de style wijzigt ofzo